app/assets/javascripts/osom-tables.js

```diff
@@ -150,14 +150,14 @@
 
   // Callback for successful ajax load of table data
   function table_load_success(container, new_content, url) {
-    var new_container = $(new_content);
-    container.html($(new_content).children());
-
-    var actual_table = new_container.find('table');
-    actual_table.data('url', url);
```

Comment:
this line is no use, the new_container is just a code piece from the internet, and it is not on this page. 
so changing the URL would not affect this page.